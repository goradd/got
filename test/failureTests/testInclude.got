package test

import (
	"bytes"
	"fmt"
)

func includeFailedTest(buf bytes.Buffer) {
{{: "blah" }}
}

