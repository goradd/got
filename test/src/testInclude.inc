
func anIncludeTest(_w io.Writer) (err error) {
{{
	Print me.
}}
    return
}

{{< aBlock }}
{{
    This is a named block.
}}
{{end aBlock}}

{{< name }}smallBlock{{end name}}