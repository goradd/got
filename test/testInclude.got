package testOut

import (
	"bytes"
	"fmt"
)

{{: "testInclude.inc" }}

func TestInclude(buf *bytes.Buffer) {
{{
The end.
}}

{{> aBlock }}

{{# the following should be found in the inc directory by the -I flag }}
{{: "testInclude2.inc" }}

{{
{{# Including an html file in text mode }}
{{: "testInclude3.html" }}
}}
}