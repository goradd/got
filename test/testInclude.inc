
func anIncludeTest(buf bytes.Buffer) {
{{
	Print me.
}}
}

{{< aBlock }}
{{
    This is a named block.
}}
{{end}}