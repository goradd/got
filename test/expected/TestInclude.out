The end.
    This is a named block.
	<p>
	Escaped html &lt;
	</p>

<p>This is text<br>
that is both escaped and has<br>
html paragraphs and breaks inserted.<br>
</p>


<html>
<head>

</head>
<body>
<p>
    A typical html document
</p>
</body>
</html>